Game/MyProject/Game/Helper/MyHelperSceneNode.h

```diff
@@ -0,0 +1,30 @@
+//-------------------------------------------------------------------------------
+//-- MyHelperSceneNode.h --------------------------------------------------------
+//-------------------------------------------------------------------------------
+#pragma once
+//-------------------------------------------------------------------------------
+//-- Dependencies ---------------------------------------------------------------
+//-------------------------------------------------------------------------------
+#include "SceneNode.hpp"
+#include "CameraSceneNode.hpp"
+//-------------------------------------------------------------------------------
+//-- Typedefs, Structs ----------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+//-------------------------------------------------------------------------------
+//-- Classes --------------------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+class MyHelperSceneNode:public SceneNode
+{
+public:
+	
+    MyHelperSceneNode(const std::string& name,RenderPacket renderPacket);
+    ~MyHelperSceneNode(){}
+    
+    void collectRenderingPackets(CameraSceneNode*,std::vector<RenderPacket>&);
+
+private:
+
+    RenderPacket       myRenderPacket;
```

Comment:
Better naming conventions and formatting please.

m_renderPacket or just renderPacket works.